local-volume/provisioner/pkg/common/common.go

```diff
@@ -58,6 +58,8 @@ const (
 	EventVolumeFailedDelete = "VolumeFailedDelete"
 	// ProvisionerConfigPath points to the path inside of the provisioner container where configMap volume is mounted
 	ProvisionerConfigPath = "/etc/provisioner/config/"
+	// ProvisionerDefaultMountRoot is used is mountDir in the config map is missing
```

Comment:
ProvisionerDefaultMountRoot is used as mountDir, if mountDir is missing from MountConfig configmap.